export.py

```diff
@@ -1,16 +1,74 @@
 #!/usr/bin/python3
 
-import json
 import os
-import re
+import json
+import re 
 import html2text
-import markdown
+import requests 
+from sys import exit as sysexit
 from bs4 import BeautifulSoup
+from getpass import getpass
 from datetime import datetime
+from markdown import markdown
 from operator import itemgetter
 from download_posts import download_posts
 from download_comments import download_comments
 
+# Generic headers to prevent LiveJournal from throwing out this random solicitation
+headers = {
+    "Upgrade-Insecure-Requests": "1",
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 OPR/113.0.0.0",
+    "sec-ch-ua": '"Chromium";v="127"',
+    "sec-ch-ua-platform": '"Windows"',
+}
+
+try:
+    # Get a "luid" cookie so it'll accept our form login.
+    cookies = {
+        'luid': requests.get("https://www.livejournal.com/", headers=headers).headers.get('Set-Cookie').split('luid=')[1].split(';')[0]
+    }
+
+except:
+    # If attempt to reach LiveJournal fails, error out.
+    print("Couldn't reach www.livejournal.com, please check internet connection. Exiting.")
```

Comment:
Here we lose the information about the exact problem, would it be helpful to keep it?

Reply:
Not exactly sure what you're asking here?

Reply:
Are you saying to capture the error from the requests, and feed to the except for reporting? Certainly could. Could even break it down into the various requests exceptions like timeouts and redirect issues.